Source/Core/UICommon/ResourcePack/ResourcePack.cpp

```diff
@@ -68,25 +69,18 @@ ResourcePack::ResourcePack(const std::string& path) : m_path(path)
   }
 
   unz_file_info manifest_info;
-
   unzGetCurrentFileInfo(file, &manifest_info, nullptr, 0, nullptr, 0, nullptr, 0);
 
-  std::vector<char> manifest_contents;
-
-  manifest_contents.resize(manifest_info.uncompressed_size);
-
+  std::string manifest_contents(manifest_info.uncompressed_size, '\0');
   if (!ReadCurrentFileUnlimited(file, manifest_contents))
```

Comment:
Does this one handle a string parameter just fine?

Reply:
It does indeed. It'f the basis for the last commit in the PR.